Avoid unnecessary dependency on libelogind.
sd_booted() always returns FALSE on Alpine.

--- a/src/core/devices/wwan/nm-modem-manager.c
+++ b/src/core/devices/wwan/nm-modem-manager.c
@@ -11,11 +11,8 @@
 
 #include <libmm-glib.h>
 
-#if HAVE_LIBSYSTEMD
-#include <systemd/sd-daemon.h>
-#else
+// XXX-Patched
 #define sd_booted() FALSE
-#endif
 
 #include "libnm-std-aux/nm-dbus-compat.h"
 #include "nm-modem.h"
